### ISSUE TITLE
NVDA-addon file extension not registered on clean installation of x64 builds

### ISSUE THREAD

**Issue by zstanecic** [nvaccess/nvda#19411]
### Brief summary

On clean windows installation, when the new x64 version of NVDA is installed, the NVDA-addon extension is not registered in the system, which makes installing NVDA add-ons from Windows explorer inpossible.

### Steps to reproduce

1. download any alpha NVDA snapshot. This applies to 2026.1 dev cycle, and even before it.
2. Install the downloaded copy. Note! It should be the clean MS Windows installation.
3. Try to locate the NVDA add-on you want to install, for example, your favourite Tts engine, or any other add-on you like, and try to install it from Windows explorer.

### Actual behavior

Windows prompt for choosing the program for opening the file extension appears. I.e Windows doesn't know which program to use to open the NVDA-addon file.

### Expected behavior

NVDA add-on installation dialog should appear with the prompt to install the add-on.

### NVDA logs, crash dumps and other attachments

Not applicable. NVDA installer doesn't create the extry for associating .nvda-addon with NVDA.

### NVDA type

installed copy

### NVDA version

alpha-53971,4a1e5c62 AMD64 and older versions of the x64 NVDA builds

### Have you tried any other versions of NVDA? If so, please report their behaviors.

Stable versions of NVDA, like 2025.1 doesn't have this problem.

### Windows version

Windows 11 25H2 (10.0.26200.7462) workstation AMD64

### Name and version of other software in use when reproducing the issue

Not applicable

### Other information about your system

Clean installation of Windows 11. seen this on other user's instances.

### Does the issue still occur after restarting your computer?

I have restarted my computer and the issue still occurs

### If NVDA add-ons are disabled, is your problem still occurring?

I do not have any NVDA add-ons installed

### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?

I have not run the COM Registration Fixing Tool
